cache-storage/src/lib.rs

```diff
@@ -77,6 +77,19 @@ impl RedisCacheStorage {
         Ok(())
     }
 
+    async fn rpushx(
```

Comment:
I am not familiar with the Redis command, so this seems like a random choice of letters to me. 

I don't think we are winning a lot by replacing 

```
 redis::cmd("RPUSHX")
            .arg(&key)
            .arg(&value)
            .query_async(&mut self.client.clone())
            .await?;
```

with 

```
cache_storate
    .rpushx(&key, &value)
    .await?;
```

I don't mean to drop this method. I ask you to think about how we can name it more like what it does and make it more meaningful.